Web/Services/AiiaService.cs

```diff
@@ -322,9 +322,14 @@ private async Task RefreshAccessToken(ApplicationUser user)
         }
         catch(AiiaClientException ex)
         {
-            if (ex.StatusCode == HttpStatusCode.Unauthorized)
+            if (ex.StatusCode == HttpStatusCode.Unauthorized || ex.StatusCode == HttpStatusCode.BadRequest)
             {
-                // The refresh token expired too.
+                // AiiaRefreshToken expiring not supported. Clearing session data.
+                user.AiiaAccessToken = null;
+                user.AiiaRefreshToken = null;
```

Comment:
do we need to clear this in the case of BadRequest?

Reply:
Good point. It got me thinking that maybe we want the new code inside the scope of the existing if-statement

```
if (ex.StatusCode == HttpStatusCode.BadRequest)
{
    user.AiiaAccessToken = null;
    user.AiiaRefreshToken = null;
    _dbContext.Users.Update(user);
    await _dbContext.SaveChangesAsync()
}
```

Reply:
Aiia returns a Bad request if the token is expired @adzspiir. Because a request to refresh is sent with an expired token.

Reply:
got it, thanks